### ISSUE TITLE
Use different icon for login and about page

### ISSUE THREAD

**Issue by veyndan** [commons-app/apps-android-commons#433]
Would it be better to use [this](https://commons.wikimedia.org/wiki/File:Commons-logo-en.svg) logo instead of the [current](https://github.com/commons-app/apps-android-commons/blob/master/app/src/main/res/drawable-hdpi/commons_logo_large.png) logo as the title is already included in the logo.

Old logo:

![device-2017-03-12-212216](https://cloud.githubusercontent.com/assets/6900601/23836095/a63d8a46-076a-11e7-881e-4dd2946c1bf0.png)

New logo:

![device-2017-03-12-212605](https://cloud.githubusercontent.com/assets/6900601/23836097/ac069d28-076a-11e7-8cc9-eb16ac6ee0eb.png)

The only problem that I would have with the new logo though is that the spacing between the title and icon is a bit too small.

**Comment by misaochan** [comment]
Honestly, the 2nd one looks a bit squashed to me, haha. What benefit would having the title in the logo confer?

**Comment by veyndan** [comment]
Yeah I agree. Mainly the title logo combo was proposed as to keep consistent with the rest of the Wikimedia Commons brand. 

I suppose better styling of the text in the first screenshot could be done. Would it be better to have the title as "Wikimedia Commons" or keep it as just "Commons"?

**Comment by misaochan** [comment]
We discussed that some time ago at #196 , and I think the conclusion was that we would use "Commons" within the app itself and "Wikimedia Commons" for external links (e.g. listings on Google Play and F Droid). So I think that means "Commons" as the title. But we are open to changes if anyone feels otherwise.  :)

**Comment by veyndan** [comment]
From reading #196, it appears that it was agreed upon that the launcher should read "Commons" for the sake of no text clipping (which is understandable). However from what I've read it doesn't seem that there was any mention as to what the name should be inside the app. Personally I would vote to use "Wikimedia Commons" everywhere (e.g. on Google Play, inside the app) but for the launcher call it "Commons".

**Comment by misaochan** [comment]
Ah, good point. I'm personally fine with either "Wikimedia Commons" or "Commons" for the login page. @nicolas-raoul what do you think?

**Comment by nicolas-raoul** [comment]
I would say the second, actually.
Better font, and that's where people need to trust the app with their
password, so better show the full name.

Cheers!
Nicolas Raoul

**Comment by misaochan** [comment]
I think veyndan is offering to "replicate" the second one by modifying the text beneath the first screenshot, but keeping the padding. How does that option sound?

**Comment by veyndan** [comment]
Any updates on which one to use. I can see the point of @nicolas-raoul so I'm favouring that atm.

**Comment by tobias47n9e** [comment]
@veyndan I think https://en.wikipedia.org/wiki/Wikipedia:Be_bold also applies to our development spirit. So if no one has strong opposition to your favored change, I think you should submit it.

**Comment by misaochan** [comment]
Haha, indeed. I'm good with 2 if everyone prefers that. :)

**Comment by misaochan** [comment]
@veyndan , any chance of the new icon being ready within the next day or so? If not, no worries, I can just put it in the next release.

**Comment by veyndan** [comment]
I am having a bit of a problem with changing the text of the icon based on the theme. A couple of solutions that I can think of is having two drawables, one for the light and one for the dark or just tint one icon.

I may get some time later on to do it.

**Comment by chrisdebian** [comment]
@veyndan Hi, Veyndan.

Not sure whether this ever got incorporated into the app; could you confirm?

Thanks,

Chris.
